services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/application/Application.kt

```diff
@@ -5,6 +5,7 @@ import fi.hel.haitaton.hanke.domain.HasId
 
 enum class ApplicationType {
     CABLE_REPORT,
+    EXCAVATION_ANNOUNCEMENT,
```

Comment:
Helsinki website calls it an excavation notification: https://www.hel.fi/en/urban-environment-and-traffic/plots-and-building-permits/permits-and-instructions-for-construction-site/working-on-streets-and-in-parks#instructions-for-excavation-work-in-street-and-park-areas

Reply:
Allu calls it excavation announcement - at least in Swagger UI: https://allu.kaupunkiymparisto.fi/external/swagger-ui/index.html#/Excavation%20announcements

Reply:
But you're right, notification it is. I will change this.